gradle/libs.versions.toml

```
@@ -3,36 +3,36 @@ metadata.format.version = "1.1"
 [versions]
 
 # API
-netty = "4.1.80.Final"
+netty = "4.1.86.Final"
 jsonSimple = "1.1.1"
 snakeyaml = "1.33"
 yamlAssist = "1.0.5"
-gson = "2.9.1"
-slf4j = "1.7.36"
-minimessage = "4.11.0"
+gson = "2.10"
+slf4j = "2.0.6"
+minimessage = "4.12.0"
 
 # General
 luckperms = "5.4"
 guava = "31.1-jre"
 floodgate = "2.2.0-SNAPSHOT"
-kotlin = "1.7.10"
+kotlin = "1.8.0"
 event = "3.0.0"
 
 # Platforms
 bukkit = "1.16.5-R0.1-SNAPSHOT"
 bungeecord = "1.19-R0.1-SNAPSHOT"
 krypton = "0.59.2"
-velocity = "3.1.2-SNAPSHOT"
+velocity = "3.2.0-SNAPSHOT"
```

Can you undo this bump please? We don't officially support Velocity, and this version seems to have changed the minimum Java version to 11, so it's not worth trying to fiddle around to support it.

```suggestion
velocity = "3.1.2-SNAPSHOT"
```